### PR TITLE
fix(links): fix links adaptation

### PR DESCRIPTION
**Pull request by xxxrenegade**
- add "new_category" language variable to all languages
- add generateAlert to no_access (alert-danger)
- add img error bootstrap alert style
- removed BG_
- selected="selected" to selected (HTMLx to HTML)
- add img-responsive to link img
- all Buttons to Bootstrap Buttons
- add generateAlert to go_back (alert-info)
- add generateAlert to no_categories (alert-info)
- new Bootstrap category issue (panel and table)
- new Bootstrap details issue (panel)
- new Bootstrap edit and new issue (panel and form-horizontal)
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23issuecomment-70417321%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23issuecomment-70720233%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131956%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131962%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131964%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23250690%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23250691%22%2C%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23250692%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23issuecomment-70417321%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Alles%20OK%20so%3F%22%2C%20%22created_at%22%3A%20%222015-01-18T17%3A39%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2109906%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/xxxrenegade%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-01-20T19%3A50%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2044e021a224e3f46e7940267785ebac5ffe09480b%20links.php%206%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131956%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Please%20use%20generateErrorBox%20instead%20of%20generateAlert%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A18%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-20T20%3A00%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20links.php%3AL34-41%22%7D%2C%20%22Pull%2044e021a224e3f46e7940267785ebac5ffe09480b%20links.php%2035%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131962%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Please%20use%20generateAlert%20here%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A18%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-20T20%3A00%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20links.php%3AL86-108%22%7D%2C%20%22Pull%2044e021a224e3f46e7940267785ebac5ffe09480b%20links.php%20114%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/webSPELL/webSPELL-4.2.3/pull/177%23discussion_r23131964%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20is%20wrong.%5Cr%5CnIf%20you%20remove%20the%20die%28%29%20you%20have%20to%20put%20the%20rest%20into%20an%20else%20case%22%2C%20%22created_at%22%3A%20%222015-01-18T09%3A19%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1415280%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Phhere%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-01-20T20%3A00%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Pascalmh%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20links.php%3AL207-214%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/Pascalmh%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/498197%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/Pascalmh'></a>

- [x] <a href='#crh-comment-Pull 44e021a224e3f46e7940267785ebac5ffe09480b links.php 6'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177#discussion_r23131956'>File: links.php:L34-41</a></b>
- <a href='https://github.com/Phhere'></a> Please use generateErrorBox instead of generateAlert
- [x] <a href='#crh-comment-Pull 44e021a224e3f46e7940267785ebac5ffe09480b links.php 35'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177#discussion_r23131962'>File: links.php:L86-108</a></b>
- <a href='https://github.com/Phhere'></a> Please use generateAlert here
- [x] <a href='#crh-comment-Pull 44e021a224e3f46e7940267785ebac5ffe09480b links.php 114'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177#discussion_r23131964'>File: links.php:L207-214</a></b>
- <a href='https://github.com/Phhere'></a> This is wrong.
If you remove the die() you have to put the rest into an else case
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177#issuecomment-70417321'>General Comment</a></b>
- <a href='https://github.com/xxxrenegade'></a> Alles OK so?

<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/177?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/177?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/webSPELL/webSPELL-4.2.3/pull/177?approve=0'></a>&nbsp;<a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by derchrisuk** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/webSPELL/webSPELL-4.2.3/pull/177'>Refresh</a>.<a href='#crh-update-none'></a>

File: links.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
 if (isset($_POST[ 'save' ])) {
     $_language->readModule('links');
     if (!ispageadmin($userID) || !isnewsadmin($userID)) {
-        die($_language->module[ 'no_access' ]);
+		echo generateAlert($_language->module[ 'no_access' ], 'alert-danger');
     }
```
Comment (Phhere) [inline links.php]: Please use generateErrorBox instead of generateAlert

File: links.php [hunk 2]
```diff
@@ -86,16 +86,22 @@
...
-                        '<strong>' . $error . '</strong><br><br>
-                        <a href="index.php?site=links&amp;action=edit&amp;linkID=' . $id . '">&laquo; ' .
-                        $_language->module[ 'back' ] . '</a>'
+                        '<div class="alert alert-danger" role="alert">
+                            <strong>' . $error . '</strong><br>
+                            <br>
+                            <a href="index.php?site=links&amp;action=edit&amp;linkID=' . $id .
+                            '" class="alert-link">&laquo; ' . $_language->module[ 'back' ] . '</a>
+                        </div>'
                     );
```
Comment (Phhere) [inline links.php]: Please use generateAlert here

File: links.php [hunk 3]
```diff
@@ -186,7 +207,7 @@
     include("_functions.php");
     $_language->readModule('links');
     if (!ispageadmin($userID) || !isnewsadmin($userID)) {
-        die($_language->module[ 'no_access' ]);
+        echo generateAlert($_language->module[ 'no_access' ], 'alert-danger');
     }
     $linkID = $_GET[ 'linkID' ];
```
Comment (Phhere) [inline links.php]: This is wrong.
If you remove the die() you have to put the rest into an else case

**Comment by xxxrenegade** [comment]
Alles OK so?

**Comment by Pascalmh** [comment]
:shipit:

File: links.php (hunk 1 above)
Comment (Pascalmh) [inline links.php]: :+1:<a href='#crh-update-none'></a>

File: links.php (hunk 2 above)
Comment (Pascalmh) [inline links.php]: :+1:<a href='#crh-update-none'></a>

File: links.php (hunk 3 above)
Comment (Pascalmh) [inline links.php]: :+1:<a href='#crh-update-complete'></a>

**Comment by Pascalmh** [comment]
Musste noch ein paar Sachen fixen nach dem merge deiner PRs https://github.com/webSPELL/webSPELL-4.2.3/commit/616a81b2bb78bf7ba78fff1e7becfbbf22f5966e

Schau mal ob dein Editor die http://editorconfig.org unterstützt (wir nutzen für php spaces, nicht tabs) und schau das die Zeilen nicht länger als 120 Zeichen lang werden. Generell nutzen wir den PSR-2 Standard.

**Pull request closed without merging**